src/modules/colorPicker/ColorPickerUI/ColorPickerUI.csproj

```diff
@@ -43,7 +43,7 @@
     <PackageReference Include="ModernWpfUI" Version="0.9.6" />
     <PackageReference Include="System.ComponentModel.Composition" Version="6.0.0" />
     <PackageReference Include="System.Data.DataSetExtensions" Version="4.5.0" />
-    <PackageReference Include="System.IO.Abstractions" Version="12.2.5" />

```

Comment:
Major version upgrades could introduce new transitive dependencies -> new .dlls we have to manually track in our Product.wxs.

Please make sure that `PowerToys.ColorPickerUI.deps.json` doesn't introduce any new stuff.

Also please do it in a separate pr.